src/Fhir.Anonymizer.Core/Extensions/FhirPathSymbolExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Hl7.Fhir.ElementModel;
+using Hl7.FhirPath.Expressions;
+
+namespace Fhir.Anonymizer.Core.Extensions
+{
+    public static class FhirPathSymbolExtensions
+    {
+        public static SymbolTable AddExtensionSymbols(this SymbolTable t)
+        {
+            t.Add("nodesByType", (IEnumerable<ITypedElement> f, string typeName) => NodesByType(f, typeName), doNullProp: true);
```

Comment:
What does the "doNullProp" mean here?

Reply:
doNullProp for null input parameters. Here we follow the same pattern as FHIR Path lib.